usr/share/rear/build/default/990_verify_rootfs.sh

```diff
@@ -3,7 +3,7 @@
 # i.e. test that the ReaR recovery system will be usable
 # to avoid issues like https://github.com/rear/rear/issues/1494
 
-LogPrint "Testing that the recovery system in $ROOTFS_DIR contains a usable system"
+LogPrint "Testing that the ReaR recovery system in '$ROOTFS_DIR' contains a usable system"
```

Comment:
```suggestion
LogPrint "Testing that the $PRODUCT recovery system in '$ROOTFS_DIR' contains a usable system"
```

Sorry, just remembered that we have a variable for that to facilitate consistency